python/packages/ai/tests/ai/planners/test_assistants_planner.py

```diff
@@ -573,6 +578,7 @@ async def test_create_azure_openai_assistant(self, mock_async_azure_openai):
 
         assistant = await AssistantsPlanner.create_assistant(
             api_key="",
+            azure_ad_token_provider=None,
```

Comment:
could we add 1-2 tests to check this flow, when its not 'None'?

Reply:
done